forte/processors/data_augment/algorithms/typo_generator.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import random
+from abc import abstractmethod
+import json
+
+
+__all__ = [
+    "TypoGenerator",
+    "UniformTypoGenerator",
+]
+
+
+class TypoGenerator:
```

Comment:
the base class doesn't seem to have an important role here besides setting the random seed. 

btw, if we want to do random, we should allow users to provide a seed so that they can expect certain output.